public/img/Vitamin-T-Logo.svg

```diff
@@ -0,0 +1,11 @@
+<svg width="160" height="36" viewBox="0 0 160 36" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0)">
```

Comment:
We should clean up the VT SVG logo by removing the <g clip-path="url(#clip0)"></g>, <clipPath id="clip0"></clipPath> and leave the path element and perhaps rect.

Another alternative you could grab the New VT logo in the dev marketing channel.